libraries/radpdfprocessing/cross-platform.md

```diff
@@ -0,0 +1,98 @@
+
+#### FixedExtensibilityManager class
+
+The new **FixedExtensibilityManager** class is exposing one property:
+
+* **FontsProvider**: Gets or sets a *FontsProviderBase* instance used to provide missing fonts.
+
+    > .NET Standard specification does not define APIs for getting specific fonts. Thus, in order to create them a custom implementation inheriting the FontsProviderBase abstract class have to be set to the FontsProvider property inside the FixedExtensibilityManager. Please, keep in mind if there is no custom implementation set a fallback to a default Font will be applied.
+
+    **Example 1** shows how you can create a custom implementation inheriting the FontsProviderBase abstract class in Windows environment.
+
+    #### **[C#] Example 1: Windows Example: Creating custom implementation inheriting the FontsProviderBase abstract class**
+    {{region cs-radpdfprocessing-cross-platform_0}}
+
+        public class FontsProvider : FontsProviderBase
+        {
+            public override byte[] GetFontData(FontProperties fontProperties)
+            {
+                if (fontProperties.FontFamilyName == "Arial")
+                {
+                    using (FileStream fileStream = File.OpenRead("arial.tff"))
+                    {
+                        using (MemoryStream memoryStream = new MemoryStream())
+                        {
+                            fileStream.CopyTo(memoryStream);
+                            return memoryStream.ToArray();
+                        }
+                    }
+                }
+                
+                return null;
+            }
+        }
+    {{endregion}}
+
+    **Example 2** shows how to set the custom implementation inheriting the FontsProviderBase abstract class to the FontsProvider property of the FixedExtensibilityManager and register a new Font to the FontsRepository dictionary.
+
+    #### **[C#] Example 2: Set the custom implementation inheriting the FontsProviderBase abstract class and register a new Font to the FontsRepository dictionary**
+    {{region cs-radpdfprocessing-cross-platform_1}}
+
+        FontsProviderBase fontsProvider = new FontsProvider();
+        FixedExtensibilityManager.FontsProvider = fontsProvider;
+
+        FontFamily fontFamily = new FontFamily("Arial");
+        FontsRepository.RegisterFont(fontFamily, new FontStyle(), new FontWeight(), fontsProvider.GetFontData(new FontProperties(fontFamily)));
+    {{endregion}}
+
+
+## Limitations in .Net Standard
+
+#### Additional settings required
+Some functionalities require additional settings to be done:
+* In order to export specific fonts, the fonts have to be registered in the FontsRepository dictionary and a custom implementation inheriting the FontsProviderBase abstract class have to be set to the FontsProvider property inside the FixedExtensibilityManager.
```

Comment:
At this point, I am with the impression that I read this sentence for the third time. Can we modify it?